notebooks/official/pipelines/google_cloud_pipeline_components_automl_tabular.ipynb

```diff
@@ -353,9 +353,9 @@
         "id": "timestamp"
```

Comment:
<p>List the steps in the pipeline and what they do.</p><p> </p><p>isplay_name="train-automl-flowers",  -&gt; update the name</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/843/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>